test/filter-tests.js

```diff
@@ -114,6 +114,11 @@ function testFilter(filename, suffix, filters) {
 // });
 
 describe('Filter test cases', function () {
+    describe('No filters', function() {
```

Comment:
Nice idea: thanks!

Reply:
Yeah, it was because my code was broken and I didn't see it :P